From 6174896e6ed938eb4dcae7f33e3829a36fb5c05a Mon Sep 17 00:00:00 2001
From: Samu Voutilainen <smar@smar.fi>
Date: Mon, 22 Apr 2013 13:03:52 -0400
Subject: [PATCH 488/625] Add comments to endifs.

This should make it easier to read construction of the makefile.

Refs #506

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Signed-off-by: Jianchuan Wang <jianchuan.wang@windriver.com>
---
 drivers/staging/lttng2/Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/lttng2/Makefile b/drivers/staging/lttng2/Makefile
index 2da7298..39672af 100644
--- a/drivers/staging/lttng2/Makefile
+++ b/drivers/staging/lttng2/Makefile
@@ -28,21 +28,21 @@ lttng-statedump-objs := lttng-statedump-impl.o wrapper/irqdesc.o \
 
 ifneq ($(CONFIG_HAVE_SYSCALL_TRACEPOINTS),)
 lttng-tracer-objs += lttng-syscalls.o probes/lttng-probe-user.o
-endif
+endif # CONFIG_HAVE_SYSCALL_TRACEPOINTS
 
 ifneq ($(CONFIG_PERF_EVENTS),)
 lttng-tracer-objs += $(shell \
 	if [ $(VERSION) -ge 3 \
 		-o \( $(VERSION) -eq 2 -a $(PATCHLEVEL) -ge 6 -a $(SUBLEVEL) -ge 34 \) ] ; then \
 		echo "lttng-context-perf-counters.o" ; fi;)
-endif
+endif # CONFIG_PERF_EVENTS
 
 obj-m += probes/
 obj-m += lib/
 
-endif
+endif # CONFIG_TRACEPOINTS
 
-else
+else # KERNELRELEASE
 	KERNELDIR ?= /lib/modules/$(shell uname -r)/build
 	PWD := $(shell pwd)
 	CFLAGS = $(EXTCFLAGS)
@@ -58,4 +58,4 @@ clean:
 
 %.i: %.c
 	$(MAKE) -C $(KERNELDIR) M=$(PWD) $@
-endif
+endif # KERNELRELEASE
-- 
2.0.2

